src/components/presentational/Layout/Layout.stories.tsx

```diff
@@ -24,34 +27,46 @@ const Template: ComponentStory<typeof Layout> = (args: LayoutProps) => <Layout {
     </Card>
 </Layout>;
 
-export const AutoColorScheme = Template.bind({});
-AutoColorScheme.args = {
-    colorScheme: "auto"
-}
-
-export const LightColorScheme = Template.bind({});
-LightColorScheme.args = {
-    colorScheme: "light"
-}
-
-export const DarkColorScheme = Template.bind({});
-DarkColorScheme.args = {
-    colorScheme: "dark"
-}
-
-export const NoGlobalStyles = Template.bind({});
-NoGlobalStyles.args = {
-    noGlobalStyles: true
-}
-
-export const CustomBackground = Template.bind({});
-CustomBackground.args = {
-    colorScheme: "dark",
-    bodyBackgroundColor: "#23395d",
-}
-
-export const CustomBackgroundLightDark = Template.bind({});
-CustomBackgroundLightDark.args = {
-    colorScheme: "auto",
-    bodyBackgroundColor: { darkMode: "#23395d", lightMode: "#ffffff" },
-}
\ No newline at end of file
+export const AutoColorScheme: Story = {
+    args: {
+        colorScheme: "auto"
+    },
+    render: render
+};
+
+export const LightColorScheme: Story = {
+    args: {
+        colorScheme: "light"
+    },
+    render: render
+};
+
+export const DarkColorScheme: Story = {
+    args: {
+        colorScheme: "dark"
```

Comment:
This doesn't seem to actually be dark, but I think that's a bug that pre-dated this PR.

Reply:
Is that a bug? Or should we just remove this story if we don't expect dark mode to affect this component?

Reply:
Dark is working for me now. 
![image](https://github.com/user-attachments/assets/424aed7a-9583-4923-b373-5b15b02d7803)